### PR TITLE
Add google tag manager

### PR DESCRIPTION
**Pull request by Akallabet**
## Description
Add GTM within the root element.
GTM will be initialised only the first time the website is loaded and not on page change.
This PR is needed in order to start testing all the different requirements from GTM console, more PRs will follow depending on feedback and change requests.

## Changelog
Description about the modification

## Checks

- [x] Implementation documented 

- [x] 80% test coverage

- [x] Ticket meets all Acceptance Criteria

## Tested in 
- [x] Chrome
- [x] Safari

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ exports.createPages = async ({graphql, actions}) => {
 
   articles.forEach(({slug, section: {slug: sectionSlug}}) => {
     const {article} = sections.find(({slug}) => slug === sectionSlug)
-    const articleList = article.map(({slug, sectionSlug, title}) => ({
```
Comment (andgreendigital) [inline gatsby-node.js]: Is this just a bug you found?

File: src/components/Analytics/Analytics.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+const tagManagerArgs = {
+  gtmId: 'GTM-T8M7L77',
+}
+
+const Analytics = () => {
+  useEffect(() => {
+    TagManager.initialize(tagManagerArgs)
+  }, [])
+  return <div />
```
Comment (andgreendigital) [inline src/components/Analytics/Analytics.js]: Why are we returning a div here?

Comment (Akallabet) [reply src/components/Analytics/Analytics.js]: humm... not sure actually, I guess we can return an empty string right?

File: gatsby-node.js (hunk 1 above)
Comment (Akallabet) [reply gatsby-node.js]: Yes, I actually fixed it during a mobbing session.
Completely unrelated of course, it fixes a bug on the SideBar component, where the links to the other articles where wrong

**Pull request merged**